src/View/XmlView.php

```diff
@@ -61,7 +61,7 @@
 class XmlView extends SerializedView
 {
     /**
-     * XML layouts are located in the xml sub directory of `Layouts/`
+     * XML layouts are located in the 'xml' sub directory of `Layouts/`
```

Comment:
Use backticks `` `xml` `` as in `Layouts` or add slash `xml/`

Reply:
And `Layouts/` are `layout/` in 4.x.

Reply:
Just use `layouts/xml/`